plasmapy/physics/transport.py

```diff
@@ -205,3 +209,1493 @@ def Coulomb_logarithm(T, n_e, particles, V=None):
     ln_Lambda = ln_Lambda.to(units.dimensionless_unscaled).value
 
     return ln_Lambda
+
+
+@check_quantity({"T_e": {"units": units.K, "can_be_negative": False},
+                 "n_e": {"units": units.m**-3},
+                 "T_i": {"units": units.K, "can_be_negative": False},
+                 "n_i": {"units": units.m**-3},
+                 })
+def classical_transport(coeffs, T_e, n_e, T_i, n_i, ion_particle, Z=None,
+                        B=0.0, model='Braginskii',
+                        field_orientation='parallel',
+                        coulomb_log_ei=None, V_ei=None,
+                        coulomb_log_ii=None, V_ii=None,
+                        hall_e=None, hall_i=None,
+                        mu=None, theta=None):
+    r"""
+    Braginskii-esque classical transport coefficients
+
+    Notes
```

Comment:
This is a neat writeup. Perhaps, since it applies to most of the functionality in this module, it could be better placed in the module's docstring at the top of this file?